lib/ngx/ssl.md

```diff
@@ -299,6 +300,43 @@ This function can be called in whatever contexts where downstream https is used.
 
 [Back to TOC](#table-of-contents)
 
+client_addr
+---------------
+**syntax:** *addr, err = ssl.client_addr()*
+
+**context:** *any*
+
+Returns the client ip address in the current SSL connection.
+
+The following code snippet shows how to delaying TLS handshake to mitigate a kind of TSL handshake attack(eg: close connection immediately after handshake complete):
+
+```lua
+local ssl = require "ngx.ssl"
+local blacklist = ngx.shared.ip_blacklist
+
+local addr, err = ssl.client_addr()
```

Comment:
It is better to handle all declared variables (especially for err) in your code. Your example will be copied by many people who want to use this method. Therefore, the code style matters.